corehq/apps/analytics/tasks.py

```diff
@@ -231,6 +231,8 @@ def track_periodic_data():
     submit = []
     for user in users_to_domains:
         email = user['email']
+        if not email:
+            continue
```

Comment:
Is this the fix we talked about? Is the fixing of `get_email`'s logic in a different PR?